src/cmd/create.go

```diff
@@ -0,0 +1,490 @@
+/*
+Copyright © 2019 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/briandowns/spinner"
+
+	"github.com/containers/toolbox/pkg/podman"
+	"github.com/containers/toolbox/pkg/utils"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	createFlags struct {
+		image   string
+		release string
+		unsafe  bool
+	}
+	ulimitHost               = []string{}
+	homeCanonical            = ""
+	homeLink                 = ""
+	mediaLink                = ""
+	mntLink                  = ""
+	toolboxProfileBind       = []string{}
+	sudoGroup                = ""
+	kcmSocket                = ""
+	kcmSocketBind            = []string{}
+	usrMountPoint            = ""
+	usrMountSourceFlags      = ""
+	usrMountDestinationFlags = "ro"
+	dbusSystemBusAddress     = ""
```

Comment:
Maybe this should be initialized with the default system bus address, which is `unix:path=/var/run/dbus/system_bus_socket` ?